perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -659,6 +659,50 @@ sub body {
     return $body;
 }
 
+=head2 time_ago
```

Comment:
It's not at the moment. I'm happy to chuck it to be honest, but might be useful elsewhere further down the line.